src/app/components/annotations/pages/search/search.component.ts

```diff
@@ -200,15 +201,19 @@ class AnnotationSearchComponent
   }
 
   protected verifyAnnotationsRoute(): StrongRoute {
-    if (this.searchParameters.routeSiteId) {
+    // I use isInstantiated() here instead of a falsy assertion so that ids of
+    // zero will pass as a valid project route id.
```

Comment:
just  note: we should never have a `0` id. It always starts from 1.

I still like the type checking aspect of this however

Reply:
Yeah, I thought about that when I made the change, but it's a minor change, and I _think_ it's more correct.